docs/provisioning/silo_vnet_newstorage.md

```diff
@@ -0,0 +1,83 @@
+# Create a silo behind a vnet with a new private storage
+
+This tutorial applies in the case you want to create a **completely new storage** account in a federated learning silo in a given region.
+
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Important: understand the design](#important-understand-the-design)
+- [Create a compute and storage pair for the silo](#need-support)
+  - [Option 1: one click deployment](#option-1-one-click-deployment)
+  - [Option 2: using az cli](#option-2-using-az-cli)
+- [Set permissions for the silo's compute to R/W from/to the orchestrator](#set-permissions-for-the-silos-compute-to-rw-fromto-the-orchestrator)
+
+## Prerequisites
```

Comment:
In the Prerequisites, would it be worth repeating what level of permissions is required?

Reply:
very good point, I applied here and also in the "open" scenario.